syntax = "proto3";

package proto;

service RuleEngine {
  rpc ExecuteRule (RuleRequest) returns (RuleResponse) {}
}

message RuleRequest {
  string fault_type = 1;
  string port = 2;
}

message RuleResponse {
  string conclusion = 1;
}
